### ISSUE TITLE
l10n_it

### ISSUE THREAD

**Issue by lap-odoo** [odoo/odoo#18563]
Dear Odoo Partners,

At Odoo, a lot of effort is put into improving the accounting localizations for each country and also to maintain these accounting localizations up-to-date. To achieve that, we will need your help.

One github issue will be created per accounting localization (e.g. l10n_be for Belgium). We expect from you to make suggestions about what should be improved in the specific accounting localization. Do not hesitate to share files with detailed explanations.

As soon as changes will be implemented, you will be informed and will have the opportunity to test it and share your feedback.

This github issue will also be useful to keep you informed about all the ongoing improvements concerning a specific accounting localization.

**Comment by eLBati** [comment]
For the sake of completeness, report here the discussion about l10n_es and odoo community/enterprise https://github.com/odoo/odoo/issues/18565#issuecomment-324260935

**Comment by sborghetti** [comment]
Hello Odoo Team @lap-odoo ,

we are members of your partner ISA SrL.
We are strongly interested to collaborate to this localization project, could you please add our GitHub account to the project team and the related repository?

Our usernames are:

s.borghetti@isa.it
a.malatesta@isa.it
v.giuliani@isa.it
d.minnocci@isa.it
i.migliorelli@isa.it

Thanks so much in advance

**Comment by lap-odoo** [comment]
Hello @sborghetti ,

Great news ! 

There is no project team but you can create one if you want (if it's easier for you to organize your work).
You can create one PR per suggested improvement/feature (example: one PR to improve CoA and one PR to add an additional accounting report). Then we will review these PR and check if it can be merged in standard. 

Keep in mind that the priority is first to have a clean CoA, Taxes, Tax Report (the basics of accounting) before starting to work on more advanced reports. 

If you have functional questions, just ping me on this issue. For the technical questions, I'll ask a colleague who is developer to help. 

Also, documentation about to create an accounting localization should be published today, it will be accessible from this page : https://www.odoo.com/documentation/11.0/index.html

Thanks for your help !

**Comment by qdp-odoo** [comment]
https://help.github.com/articles/cloning-a-repository/

merry christmas

**Comment by qdp-odoo** [comment]
or maybe https://help.github.com/

**Comment by lap-odoo** [comment]
Hi @sborghetti 

Yes, this is what I suggest. If you have improvements in mind but you're not sure they will be accepted, we can also discuss them here, before you start working on it. 

Here is the l10n_it module (community) : https://github.com/odoo/odoo/tree/11.0/addons/l10n_it
You can also functionally test it on our runbot : http://runbot.odoo.com/runbot (admin, admin) 

Your changes should target the Master branch (not the V11). 

The previous links sent by @qdp-odoo will be helpful too.

**Comment by eLBati** [comment]
Hello @lap-odoo, @qdp-odoo and @sborghetti 

a brief state of the art:
1. **l10n_it module**
 It contains Italian chart of accounts and chart of taxes. It can be improved, like removing old and not used anymore taxes, removing 'reconcile' bool from accounts that must not be reconciled, improving account types  and tags, in case
2. **OCA Italian localization modules**
 As you probably know, an OCA repository for Italian localization exists at https://github.com/OCA/l10n-italy and contains many localization modules yet (v10). Among them, [l10n_it_vat_registries](https://github.com/OCA/l10n-italy/tree/10.0/l10n_it_vat_registries) is the base tax report required by law

Next steps, in the first place
1. **Improve l10n_it module**
 This can only be done by us (members of the Italian community) I think, by PR against ```l10n_it``` module (```master``` or ```11.0```?)
2. **Include tax report in odoo**
 This should be done by odoo guys and Italian community guys together and the first goal should be to include ```l10n_it_vat_registries``` module in odoo. So I invite @qdp-odoo to take a look at it.
 The module also depends on ```account_tax_balance```, another OCA module, that introduces a functionality that, AFAIR, is provided by odoo enterprise: compute taxes balances given a date range.

That being said, I will be happy to help to include ```l10n_it_vat_registries```, and other modules hopefully, into odoo core, only if they will be licensed under LGPL.
Otherwise, why should Italian companies invest time (in addition to what already done to develop OCA/l10n-italy) in modules that will be sold by Odoo SA?

I we agree on the next steps and the policy, let's start!

**Comment by lap-odoo** [comment]
Hello @eLBati 

1. You can do a PR that targets the master.
2.  This report could be included in Odoo Enterprise (not Odoo Community). As explained [here](https://github.com/odoo/odoo/issues/18565), Odoo SA doesn't support anymore the accounting in the community version (even if it can supported thanks to the addition of community modules). 

To answer to your question "why should Italian companies invest time (in addition to what already done to develop OCA/l10n-italy) in modules that will be sold by Odoo SA?", if you are selling Odoo Enterprise to your customers you have an interest in having an Italian localization package that is complete and working out of the box.

**Comment by eLBati** [comment]
> you have an interest in having an Italian localization package that is complete and working out of the box

@lap-odoo 
We already have such a package: it is composed by OCA/l10n-italy modules + proprietary modules developed and maintained by [Odoo Italia Network](https://www.odoo-italia.net) (btw: a tax report is not enough to build an Italian localization package that is complete and working out of the box).

**Comment by lap-odoo** [comment]
@eLBati : I agree but it is not officially supported and maintained by Odoo SA. If features are included into official Odoo SA modules, we take care of the bug fixing, of the updates that may be necessary and we upgrade the module from one version to the other. That being said, It's your choice to decide if you want to help or not knowing that, as explained, all the features/data related to invoicing will be in the community module and all the features/data related to accounting will be in the enterprise module.

**Comment by jco-odoo** [comment]
We created the necessary requirements for the Italian e-invoicing (branch: https://github.com/odoo-dev/odoo/tree/12.0-ediit-jco  )  We appreciate any help with testing out as we need people with a VAT number(+codice fiscale) and a PEC-mail.

**Comment by avw-odoo** [comment]
Dear Odoo Partners,

We are working on the Liquidazione IVA.
We have some questions regarding the report itself and the tax return process, could you help us ?

### About the vat return process in accounting
- which accounts are involved in the tax return closing (vat centralization) ?
  - If we pay the vat due to the administration
  - If the administration pay us
  - If we pay advance amounts

### About the tax grid
([Official specification](https://www.agenziaentrate.gov.it/portale/documents/20143/289220/Form+and+instructions+-+Periodic+Vat+settlements+communication_IVA+PERIODICA+2018_EN.PDF/cae2dd82-5be7-540d-96ff-1f41e186c79d))

In the context of these questions, a year always start at January 1st and ends on December 31th. Unless specified otherwise, mentioned fields are for the reference period.
- A period can be a quarter (three months, ex.: Jan, Feb and Mar) or a month (ex.: Jan) ? Can it be something else ? Does it make sense to be something else ?
- VP7
  - Can we put exactly 25.82€ in the VP7 field ?
  - Is this field always zero for the first period of the year ?
  - [reference period is not the first of the year] Is it mandatory for the taxpayer to fill field VP7 if he had less than 25.82€ of debt in the last period ?
  - Can the taxpayer pay a vat return that is less than 25.82€ in the reference period ? Ex.: He has 20€ of debt in the reference period, he pays the state 20€ and in the next period, he indicates 0€ in VP7.
- VP8 
  - Is this field always zero for the first period of the year ?
  - Is it equal to the amount indicated in "VP14 (credit)" of the last period, given the last period occurs during the same year.
  - Can it be greater than VP6 (debt) ? Can VP14 be in credit as a result of VP8 being greater than debts ?
- VP9
  - Is VP9 always zero except for the first period of the year ?
  - Can VP9 be greater than VP6 (debt) ?
  - [last year last period resulted in VP14 (credit) being positive] Can VP9 of the first period of the year equal to the full amount of last year last period VP14 (credit) regardless of the amount of debt or credit for this first period ?
  - [reference period is the first period of the year] Is there any incentive not to indicate the full amount of last year last period VP14 (credit) ?
  - Can VP9 be strictly less than previous year last period VP14 (credit) for any given period ?
  - Can VP9 be strictly greater than previous year last period VP14 (credit) for any given period ?
  - Is it possible to claim credit through model F24 without the claimed amount first appearing in the field VP9, regardless of the period VP9 relates to ?
  - If I claimed 3000€ through model F24 during the reference period, do I have to indicate the amount -3000€ (minus three thousand euro) in the field VP9 ?
- VP10
  - I buy 3 000 000€ of German (any euro country not being Italy) cars, pay the full amount along with 600 000€ of VAT and I don't sell any of these cars in the reference period, will the 600 000€ VAT amount appear anyway in my VAT report ?
  - I bought a German car three months ago, 30 000€ (base) + 6000€ (vat), and I now sell this car, and only this car, will VP10 equal 6000€ ? Will the base amount accounts for any field of the reference period VAT report ?
- VP11
  - Where does this field takes its input from ?
- VP12
  - Do a lot of taxpayers choose to do quarterly vat returns ? In the affirmative, which ones, or why ?
  - Do only quarterly taxpayers have to pay this ?
  - Does any taxpayer has to fill VP12 for the last quarter of the year ?
  - What is the basis the 1% interest rate is applied onto ?
  - How is computed the 1% interest rate ? 0.01 * basis ?
- VP13
  - Does this field concern many taxpayer or only very specific ones ?
  - In the case the taxpayer made a prepayment for the reference period, did it occur during the reference period or can it occur before the reference period ?
  - Does the field VP13 indicates exactly the amount paid as a prepayment ?
- VP14
  - Is it frequent of this field to result in a credit ?

Thank you for your help.

cc @AdrienHorgnies

**Comment by jco-odoo** [comment]
@eLBati

**Comment by eLBati** [comment]
@avw-odoo @jco-odoo hi, the Liquidazione IVA module will be released under Odoo Community?

**Comment by jco-odoo** [comment]
@eLBati With v13, the data about the tax report lines were put in community, although the report is still enterprise.  The problem here however is that the report asks for numbers about previous periods, ... which makes us (for the moment) fall back on the old financial report lines.

**Comment by C3POdoo** [comment]
Dear @lap-odoo,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
